src/Windows/Avalonia.Win32/Avalonia.Win32.csproj

```diff
@@ -4,6 +4,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- We still keep BinaryFormatter for WinForms compatibility. -->
     <EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>
+    <BuiltInComInteropSupport>false</BuiltInComInteropSupport>
```

Comment:
Even with net6?

Reply:
Left-over, will remove. It doesn't seem to have effect on library projects either way.